### ISSUE TITLE
Use of submodules

### ISSUE THREAD

**Issue by lacombar** [JBakamovic/yavide#25]
The install script logic duplicate the git-submodule functionality. This is unnecessarily complicated.

**Comment by JBakamovic** [comment]
I disagree with this one. I can't really see what would be the advantages of using `git-submodule` over `git-clone`/`git-pull` mechanism. Power of `git-submodule` comes into play if one wants to customize the external dependencies to fit into the project and afterwards merge the upstream changes. At this moment, this is not the case with `Yavide`. There is no customization of external dependencies.
